packages/server/src/profile/profile.controller.ts

```diff
@@ -177,6 +180,67 @@ export class ProfileController {
       }
     });
   }
+
+  @Get(':id/user')
+  @UseGuards(JwtAuthGuard)
+  async getUser(@Param('id') id: number): Promise<UserModel> {
+    const user = await UserModel.findOne(id);
+    return user;
+  }
+
+  @Post(':id/edit_user')
+  @UseGuards(JwtAuthGuard)
+  async edit_user(
+    @Param('id') id: number,
+    @Body() body: UBCOuserParam,
+  ): Promise<EditUserResponse> {
+    const user = await UserModel.findOne(id);
+
+    if (!user) {
```

Comment:
I think we should add a check to verify that person with right permission can do this operation.